target/linux/ar71xx/base-files/lib/upgrade/platform.sh

```diff
@@ -575,7 +578,11 @@ platform_check_image() {
 	eap120|\
 	wbs210|\
 	wbs510)
-		tplink_pharos_check_image "$1" && return 0
+		tplink_pharos_check_image "$1" "7f454c46" "$(tplink_pharos_get_model_string)" '' && return 0
+		return 1
+		;;
+	cpe210-v2)
+		tplink_pharos_v2_check_image "$1" "01000000" "$(tplink_pharos_v2_get_model_string)" '\0\xff\r' && return 0
```

Comment:
Sry my stupped question - where is `tplink_pharos_v2_check_image` could only find `tplink_pharos_check_image`

Reply:
Good find. I overlooked this when merging tplink_pharos_v2_check_image and tplink_pharos_check_image.
Commit should already be updated!